src/TreeView/TreeView.test.tsx

```diff
@@ -983,13 +990,15 @@ describe('Asyncronous loading', () => {
     // Wait for async loading to complete
     const firstChild = await findByRole('treeitem', {name: 'Child 1'})
 
-    setTimeout(() => {
-      // First child should be focused
-      expect(firstChild).toHaveFocus()
+    act(() => {
+      jest.runAllTimers()
     })
+
+    // First child should be focused
+    expect(firstChild).toHaveFocus()
   })
 
-  it.only('moves focus to parent item after closing error dialog', async () => {
+  it.skip('moves focus to parent item after closing error dialog', async () => {
```

Comment:
Is this test not working? Curious why we are skipping it.

Reply:
@colebemis it seems like something is going on with it in `main` 🤔 

When using `it.only` it passes, when removing `it.only` the test ends up failing. Not sure what's going on with it.

Reply:
I resolved it in the branch, I think it should be okay now!